### ISSUE TITLE
Crash in block editor on double rotation when inserter is open

### ISSUE THREAD

**Issue by hypest** [wordpress-mobile/WordPress-Android#10227]
### Steps to reproduce the behavior

1. Start a new post in the block editor
2. Tap on the "+" Inserter button and let the Inserter open
3. Rotate the device
4. Rotate back
5. 💥

##### Tested on [device], Android [version], WPAndroid [version]

Google Pixel 2XL, Android 9, WPAndroid v12.9-rc-1 (though is probably happening in older versions too)

**Comment by daniloercoli** [comment]
Reproduced the crash on a very old version of the app which had GB mobile in it (alpha-150).

**Comment by daniloercoli** [comment]
The first rotation does already show a problem:
```
2019-07-16 12:13:36.517 5392-5392/org.wordpress.android E/WindowManager: android.view.WindowLeaked: Activity org.wordpress.android.ui.posts.EditPostActivity has leaked window DecorView@d102006[EditPostActivity] that was originally added here
        at android.view.ViewRootImpl.<init>(ViewRootImpl.java:701)
        at android.view.WindowManagerGlobal.addView(WindowManagerGlobal.java:365)
        at android.view.WindowManagerImpl.addView(WindowManagerImpl.java:93)
        at android.app.Dialog.show(Dialog.java:470)
        at com.facebook.react.views.modal.ReactModalHostView.showOrUpdate(ReactModalHostView.java:266)
        at com.facebook.react.views.modal.ReactModalHostManager.onAfterUpdateTransaction(ReactModalHostManager.java:108)
        at com.facebook.react.views.modal.ReactModalHostManager.onAfterUpdateTransaction(ReactModalHostManager.java:29)
        at com.facebook.react.uimanager.ViewManager.updateProperties(ViewManager.java:47)
        at com.facebook.react.uimanager.NativeViewHierarchyManager.createView(NativeViewHierarchyManager.java:268)
        at com.facebook.react.uimanager.UIViewOperationQueue$CreateViewOperation.execute(UIViewOperationQueue.java:198)
        at com.facebook.react.uimanager.UIViewOperationQueue$1.run(UIViewOperationQueue.java:838)
        at com.facebook.react.uimanager.UIViewOperationQueue.flushPendingBatches(UIViewOperationQueue.java:952)
        at com.facebook.react.uimanager.UIViewOperationQueue.access$2200(UIViewOperationQueue.java:44)
        at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.doFrameGuarded(UIViewOperationQueue.java:1012)
        at com.facebook.react.uimanager.GuardedFrameCallback.doFrame(GuardedFrameCallback.java:29)
        at com.facebook.react.modules.core.ReactChoreographer$ReactChoreographerDispatcher.doFrame(ReactChoreographer.java:172)
        at com.facebook.react.modules.core.ChoreographerCompat$FrameCallback$1.doFrame(ChoreographerCompat.java:84)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:947)
        at android.view.Choreographer.doCallbacks(Choreographer.java:761)
        at android.view.Choreographer.doFrame(Choreographer.java:693)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:935)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:6986)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1445)
```

and then the crash happens on the 2nd rotation
```
2019-07-16 12:13:56.989 5392-5392/org.wordpress.android E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.wordpress.android, PID: 5392
    java.lang.RuntimeException: Unable to resume activity {org.wordpress.android/org.wordpress.android.ui.posts.EditPostActivity}: java.lang.IllegalArgumentException: View=DecorView@d102006[EditPostActivity] not attached to window manager
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3993)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4025)
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1926)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:6986)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1445)
     Caused by: java.lang.IllegalArgumentException: View=DecorView@d102006[EditPostActivity] not attached to window manager
        at android.view.WindowManagerGlobal.findViewLocked(WindowManagerGlobal.java:533)
        at android.view.WindowManagerGlobal.updateViewLayout(WindowManagerGlobal.java:419)
        at android.view.WindowManagerImpl.updateViewLayout(WindowManagerImpl.java:99)
        at android.app.Dialog.onWindowAttributesChanged(Dialog.java:914)
        at android.view.Window.dispatchWindowAttributesChanged(Window.java:1134)
        at com.android.internal.policy.PhoneWindow.dispatchWindowAttributesChanged(PhoneWindow.java:3169)
        at android.view.Window.setFlags(Window.java:1111)
        at android.view.Window.clearFlags(Window.java:1085)
        at com.facebook.react.views.modal.ReactModalHostView.updateProperties(ReactModalHostView.java:304)
        at com.facebook.react.views.modal.ReactModalHostView.showOrUpdate(ReactModalHostView.java:211)
        at com.facebook.react.views.modal.ReactModalHostView.onHostResume(ReactModalHostView.java:175)
        at com.facebook.react.bridge.ReactContext.onHostResume(ReactContext.java:208)
        at com.facebook.react.ReactInstanceManager.moveToResumedLifecycleState(ReactInstanceManager.java:661)
        at com.facebook.react.ReactInstanceManager.onHostResume(ReactInstanceManager.java:580)
        at com.facebook.react.ReactInstanceManager.onHostResume(ReactInstanceManager.java:536)
        at org.wordpress.mobile.WPAndroidGlue.WPAndroidGlueCode.onResume(WPAndroidGlueCode.java:342)
        at org.wordpress.android.editor.GutenbergContainerFragment.onResume(GutenbergContainerFragment.java:84)
        at androidx.fragment.app.Fragment.performResume(Fragment.java:2498)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1501)
        at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1784)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1852)
        at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange(FragmentManager.java:3269)
        at androidx.fragment.app.FragmentManagerImpl.dispatchResume(FragmentManager.java:3241)
        at androidx.fragment.app.Fragment.performResume(Fragment.java:2504)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1501)
        at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1784)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1852)
        at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange(FragmentManager.java:3269)
        at androidx.fragment.app.FragmentManagerImpl.dispatchResume(FragmentManager.java:3241)
        at androidx.fragment.app.FragmentController.dispatchResume(FragmentController.java:223)
        at androidx.fragment.app.FragmentActivity.onResumeFragments(FragmentActivity.java:538)
2019-07-16 12:13:56.990 5392-5392/org.wordpress.android E/AndroidRuntime:     at androidx.fragment.app.FragmentActivity.onPostResume(FragmentActivity.java:527)
        at androidx.appcompat.app.AppCompatActivity.onPostResume(AppCompatActivity.java:172)
        at android.app.Activity.performResume(Activity.java:7582)
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3985)
        	... 11 more
```

**Comment by daniloercoli** [comment]
Another flow that reproduce the problem:
- Start a new post in the block editor
- Tap on the "+" Inserter button and let the Inserter open
- Rotate the device
- Tap back and close the Inserter
- Rotate the device back
- 💥

**Comment by etoledom** [comment]
Testing on a Xiaomi Mi A2 Lite:

It's interesting that rotating the device shows the keyboard even when there are no blocks selected.
And it sometimes(?) focuses the PostTitle.

Might it be a conflict between however the modal works on Android and the Keyboard?

**Comment by mkevins** [comment]
I have discovered the source of this crash, as well as the leaked context (great observation regarding the leak @daniloercoli :+1: ) There is a fix for the crash here: https://github.com/wordpress-mobile/WordPress-Android/pull/10383.

@etoledom you are quite right about the strange keyboard behavior, and I suspect is has something to do with focus behavior during the initialization of the editor on the second go-around. This may also be related to the strange behavior described here: https://github.com/wordpress-mobile/WordPress-Android/pull/10383#issuecomment-521123642 after the fix is applied.

**Comment by cameronvoell** [comment]
I'm working on this issue now. I'm working off of @mkevins insight and some new details listed in #10383 and [gb-mobile issue 1333](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1333)

**Comment by mkevins** [comment]
> I just verified that this issue seems isolated to newest Android version, Pie/ API 28 /v9.0.

Nice catch, @cameronvoell ! If nothing stands out in the Android code, I wonder whether `react-native-modal` could have some version specific logic, or is somehow not working as expected as a result of the Android changes.

Do you know whether the leaked context was occurring on the older API versions? That may give a clue to the underlying cause as well, particularly if life-cycle events are somehow handled differently in the newer API.

**Comment by etoledom** [comment]
`react-native-modal` package is a simple component based on the React Native `Modal` component: https://github.com/facebook/react-native/blob/master/Libraries/Modal/Modal.js

If the bug is on React Native side, it might be worth it to check: https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/views/modal/ReactModalHostView.java

**Comment by cameronvoell** [comment]
> Do you know whether the leaked context was occurring on the older API versions?

@mkevins  So testing on PRE API 28, there is no leaked context error message in the **Android Logcat log**. On API 28 I am seeing the same messages [@daniloercoli reported in the previous comment](https://github.com/wordpress-mobile/WordPress-Android/issues/10227#issuecomment-511756539).

There is a message however in the Javascript logs referencing memory leaks. However this message shows before and after your fix is implemented on API PRE 28 and on API 28. So it does not cause the crash alone. But it could be related

`2019-09-01 15:55:13.373 17363-18677/org.wordpress.android E/WordPress-EDITOR: 'Warning: Can\'t perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in %s.%s', 'the componentWillUnmount method', '\n    in SlotComponent (at slot.js:95)\n    in Slot (at slot-fill/index.js:16)\n    in InspectorControlsSlot (at block-mobile-toolbar.native.js:31)\n    in RCTView (at View.js:35)\n    in View (at block-mobile-toolbar.native.js:26)\n    in BlockMobileToolbar (at with-dispatch/index.js:98)\n    in WithDispatch(BlockMobileToolbar) (at with-select/index.js:60)\n    in Unknown (at pure/index.js:38)\n    in WithSelect(WithDispatch(BlockMobileToolbar)) (at block.native.js:137)`


> If the bug is on React Native side, it might be worth it to check: https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/views/modal/ReactModalHostView.java

@etoledom So debugging the two apps, one running on API 28 and one pre 28, the clearest divergence I see happening does seem to be within the ReactModalHostView [here](https://github.com/facebook/react-native/blob/7b35f427fd66cb0f36921b992095fe5b3c14d8b9/ReactAndroid/src/main/java/com/facebook/react/views/modal/ReactModalHostView.java#L327).

That line runs fine on pre API 28, but it is crashing on API 28. Unfortunately, I think this is caused by some race condition between the WindowManager, the restarting Activity, and the ReactModalHostView and its Dialog object. There are hints that ReactModalHostView has had similar issues in the past in React Native update notes, one of which points to this fix, which has the same error message that we are seeing (java.lang.IllegalArgumentException): https://github.com/facebook/react-native/commit/e360b0b.

I think there is still a good chance that we could fix the behavior of API 28 so that it behaves like the non crashing, modal preserving Pre-API 28 behavior shown in my GIF above. However it is hard to estimate the time for a fix since it could be an issue with the React Native modal, or somehow related to how we are updating context, and references to activities within our "headless" retained fragment setup, containing our React Native Root view. #10383 could be a good short term fix for at least not crashing. This brings me to my last point:

> It's interesting that rotating the device shows the keyboard even when there are no blocks selected.
> And it sometimes(?) focuses the PostTitle.

There seems to be some existing issues in the editor that show the wrong blocks having focus on rotate, that is actually unrelated to Modals. I'll upload a GIF in a separate issue and link to it here. Even with no modal open, rotating the phone can sometimes select an incorrect text block and then open a keyboard over the screen unexpectedly. This in my opinion is just as bad or worse of an issue than this one, since it does not require a modal to be open, and it would quite jarring for a user. Fixing scroll position and caret position was documented and fixed earlier according to this https://github.com/wordpress-mobile/WordPress-Android/issues/8739, so I think incorrect focus of blocks on rotate could probably be debugged as a regression, and I'd welcome any advice on when the team thinks the bad behavior could have been introduced. 

## Suggested Steps:
1. Address Regression of incorrect blocks gaining focus as a separate issue (will link here)
2. Spend another day or two trying to get API 28 to behave like pre API 28.
3. In Parallel to (2.) work on solving side effects of #10383 . (Modals should be destroyed on close, not hidden, and we need to investigate the [incorrect Add Block mkevins found](https://github.com/wordpress-mobile/WordPress-Android/pull/10383#issuecomment-521123642)) 
4. As a longer term solution for rotation issues, we should perhaps revisit the idea of not destroying the EditPostActivity on rotate, and any refactoring to accomodate that, which was referenced as the plan 8 months ago. This seems to be reliant on sunsetting Aztec, however.  See https://github.com/wordpress-mobile/WordPress-Android/pull/8800, https://github.com/wordpress-mobile/WordPress-Android/pull/8874, and https://github.com/wordpress-mobile/WordPress-Android/pull/9030.

**Comment by mkevins** [comment]
Hi @cameronvoell :wave: 

> So testing on PRE API 28, there is no leaked context error message

Thanks for checking that. I think this suggests a difference (pre/post API 28) in how the context is handled (perhaps either by RN itself, or the `react-native-modal` library).

> There is a message however in the Javascript logs referencing memory leaks.

IIRC, this isn't always Android specific (but it can be), and in this case seems related to the block toolbar, so I wouldn't think it's related to the modal.

> we need to investigate the incorrect Add Block mkevins found

Some more info on that: When the inserter menu is closed, it usually hides the "insertion point" here: https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/inserter/menu.native.js#L60 . So, this is either not being called, or somehow the saved state of the app is causing it to be "un-hidden" somehow when the activity is recreated. I haven't investigated this deeply, so I'm only noting this here for now in case it's useful.

**Comment by mzorz** [comment]
Came down here tracking another issue - this crash is being tracked in Sentry, adding that information here:
 
Comes from https://github.com/wordpress-mobile/WordPress-Android/issues/9494#issuecomment-540833647

**90-day impact:** ~16 per day
**Users affected in the last 90 days:** 626
**Limited to:** Android 9 and above

<sup>https://sentry.io/share/issue/136cee38cb79491db456071ee0c686a0/</sup>

**Comment by designsimply** [comment]
**90-day impact:** ~19 per day
**Users affected in the last 90 days:** 695
**Limited to:** Android 9 and above
<sup>https://sentry.io/share/issue/136cee38cb79491db456071ee0c686a0/</sup>

**Comment by designsimply** [comment]
**Users affected in the last 90 days:** 768
<sub>https://sentry.io/share/issue/136cee38cb79491db456071ee0c686a0/</sub>

**Comment by designsimply** [comment]
Tested and confirmed using WPAndroid 13.8-rc-1 on Pixel 3 Android 10.

Steps to reproduce:

1. Add and image block.
1. Tap "Add Image".
1. Rotate the device and rotate it back again.

**Result:** the editor closes automatically if you rotate the device while the add image BottomSheet is open. ([15s](https://cloudup.com/c1xCdDI-RHM))

<details><summary>Click to expand app logs.<br>
WordPress - 13.8-rc-1 - Version code: 802<br>
Android device name: Google Pixel 3</summary>

01 - [Dec-07 06:53 API] Dispatching action: PostAction-PUSH_POST
02 - [Dec-07 06:53 UTILS] Connection status changed, isConnected=true
03 - [Dec-07 06:53 NOTIFS] notifications update job service > created
04 - [Dec-07 06:53 STATS] 🔵 Tracked: notifications_upload_post_error_retry
05 - [Dec-07 06:53 NOTIFS] notifications update job service > destroyed
06 - [Dec-07 06:53 NOTIFS] notifications update job service > created
07 - [Dec-07 06:53 API] Dispatching action: AccountAction-FETCHED_ACCOUNT
08 - [Dec-07 06:53 API] Dispatching action: AccountAction-FETCHED_SETTINGS
09 - [Dec-07 06:53 API] Dispatching action: SiteAction-UPDATE_SITE
10 - [Dec-07 06:53 API] Dispatching action: SiteAction-FETCHED_SITE_EDITORS
11 - [Dec-07 06:53 DB] Site found by (local) ID: 71
12 - [Dec-07 06:53 DB] Updating site: http://made4testing0318.blog
13 - [Dec-07 06:53 DB] Site found by (local) ID: 71
14 - [Dec-07 06:53 DB] Updating site: http://made4testing0318.blog
15 - [Dec-07 06:53 API] Dispatching action: PostAction-FETCHED_POST_LIST
16 - [Dec-07 06:53 API] Dispatching action: ListAction-FETCHED_LIST_ITEMS
17 - [Dec-07 06:53 API] Dispatching action: PostAction-FETCHED_POST_LIST
18 - [Dec-07 06:53 API] Dispatching action: ListAction-FETCHED_LIST_ITEMS
19 - [Dec-07 06:53 API] Dispatching action: SiteAction-FETCHED_SITES
20 - [Dec-07 06:53 DB] Site found by SITE_ID: 159889361
21 - [Dec-07 06:53 DB] Updating site: http://made4testing0318.blog
22 - [Dec-07 06:53 DB] Site found by SITE_ID: 160991665
23 - [Dec-07 06:53 DB] Updating site: http://gotravelrewards.blog
24 - [Dec-07 06:53 DB] Site found by SITE_ID: 161935399
25 - [Dec-07 06:53 DB] Updating site: http://fashionsite.fashion.blog
26 - [Dec-07 06:53 DB] Site found by SITE_ID: 160502386
27 - [Dec-07 06:53 DB] Updating site: http://artindesignco.blog
28 - [Dec-07 06:53 DB] Site found by SITE_ID: 169070507
29 - [Dec-07 06:53 DB] Updating site: https://crooked-badger.jurassic.ninja
30 - [Dec-07 06:53 DB] Site found by SITE_ID: 5276229
31 - [Dec-07 06:53 DB] Updating site: https://design5279.wordpress.com
32 - [Dec-07 06:53 DB] Site found by SITE_ID: 160805144
33 - [Dec-07 06:53 DB] Updating site: http://wibblywobblytimeywimey.travel.blog
34 - [Dec-07 06:53 DB] Site found by SITE_ID: 160948467
35 - [Dec-07 06:53 DB] Updating site: http://lovinglycompetitivestorytelling.news.blog
36 - [Dec-07 06:53 DB] Site found by SITE_ID: 159662180
37 - [Dec-07 06:53 DB] Updating site: https://madefortesting190313.wordpress.com
38 - [Dec-07 06:53 DB] Site found by SITE_ID: 160991226
39 - [Dec-07 06:53 DB] Updating site: https://musictosingoutloud.wordpress.com
40 - [Dec-07 06:53 DB] Site found by SITE_ID: 160808369
41 - [Dec-07 06:53 DB] Updating site: https://myfiadventures.wordpress.com
42 - [Dec-07 06:53 DB] Site found by SITE_ID: 160805604
43 - [Dec-07 06:53 DB] Updating site: https://myrandomyoutubevideos.wordpress.com
44 - [Dec-07 06:53 DB] Site found by SITE_ID: 161701929
45 - [Dec-07 06:53 DB] Updating site: http://thetimeywimeykindof.family.blog
46 - [Dec-07 06:53 DB] Site found by SITE_ID: 161693475
47 - [Dec-07 06:53 DB] Updating site: http://youtubevideosilike.movie.blog
48 - [Dec-07 06:53 API] Dispatching action: AccountAction-FETCHED_SUBSCRIPTIONS
49 - [Dec-07 06:53 NOTIFS] notifications update service > completed
50 - [Dec-07 06:53 NOTIFS] notifications update job service > all tasks completed
51 - [Dec-07 06:53 NOTIFS] notifications update job service > destroyed
52 - [Dec-07 06:53 API] Dispatching action: UploadAction-PUSHED_POST
53 - [Dec-07 06:53 API] Dispatching action: PostAction-PUSHED_POST
54 - [Dec-07 06:53 API] Dispatching action: ListAction-LIST_ITEMS_CHANGED
55 - [Dec-07 06:53 API] Dispatching action: ListAction-LIST_DATA_INVALIDATED
56 - [Dec-07 06:53 API] Dispatching action: ListAction-LIST_REQUIRES_REFRESH
57 - [Dec-07 06:53 API] Dispatching action: PostAction-FETCH_POST_LIST
58 - [Dec-07 06:53 API] Dispatching action: PostAction-FETCH_POST_LIST
59 - [Dec-07 06:53 API] Dispatching action: ListAction-LIST_DATA_INVALIDATED
60 - [Dec-07 06:53 UTILS] SnackbarSequencer > New item added
61 - [Dec-07 06:53 UTILS] SnackbarSequencer > invoking start()
62 - [Dec-07 06:53 API] Dispatching action: ListAction-LIST_DATA_INVALIDATED
63 - [Dec-07 06:53 POSTS] PostUploadHandler - DO_NOTHING. Post: 
64 - [Dec-07 06:53 UTILS] SnackbarSequencer > prepareSnackBar message [Draft saved online]
65 - [Dec-07 06:53 UTILS] SnackbarSequencer > before delay
66 - [Dec-07 06:53 POSTS] PostUploadHandler > Completed
67 - [Dec-07 06:53 API] Dispatching action: PostAction-FETCHED_POST_LIST
68 - [Dec-07 06:53 API] Dispatching action: ListAction-FETCHED_LIST_ITEMS
69 - [Dec-07 06:53 API] Dispatching action: PostAction-FETCHED_POST_LIST
70 - [Dec-07 06:53 API] Dispatching action: ListAction-FETCHED_LIST_ITEMS
71 - [Dec-07 06:53 UTILS] SnackbarSequencer > after delay
72 - [Dec-07 06:53 UTILS] SnackbarSequencer > item removed from the queue
73 - [Dec-07 06:53 UTILS] SnackbarSequencer > finishing start()
74 - [Dec-07 06:53 API] QuickStartStore onRegister
75 - [Dec-07 06:53 STATS] 🔵 Tracked: deep_link_not_default_handler, Properties: {"interceptor_classname":"com.android.internal.app.ResolverActivity"}
76 - [Dec-07 06:53 UTILS] trackLastActivity, activityId: My Site
77 - [Dec-07 06:53 STATS] 🔵 Tracked: my_site_tab_accessed, Properties: {"blog_id":159889361,"is_jetpack":false}
78 - [Dec-07 06:53 API] Dispatching action: AccountAction-FETCH_ACCOUNT
79 - [Dec-07 06:53 PROFILING] App Startup: begin
80 - [Dec-07 06:53 PROFILING] App Startup: 8332 ms, WPMainActivity.onCreate
81 - [Dec-07 06:53 PROFILING] App Startup: 80 ms, WPMainActivity.onResume
82 - [Dec-07 06:53 PROFILING] App Startup: end, 8412 ms
83 - [Dec-07 06:53 API] Dispatching action: SiteAction-FETCH_POST_FORMATS
84 - [Dec-07 06:53 API] Dispatching action: SiteAction-FETCH_SITE_EDITORS
85 - [Dec-07 06:53 NOTIFS] Sending GCM token to our remote services: *** REDACTED ***
86 - [Dec-07 06:53 API] Received site Categories
87 - [Dec-07 06:53 API] Successfully fetched WP.com categories
88 - [Dec-07 06:53 API] Received response to Settings REST request.
89 - [Dec-07 06:53 API] Dispatching action: AccountAction-FETCHED_ACCOUNT
90 - [Dec-07 06:53 API] Dispatching action: SiteAction-FETCHED_POST_FORMATS
91 - [Dec-07 06:53 API] Dispatching action: SiteAction-FETCHED_SITE_EDITORS
92 - [Dec-07 06:53 DB] Site found by (local) ID: 71
93 - [Dec-07 06:53 DB] Updating site: http://made4testing0318.blog
94 - [Dec-07 06:53 NOTIFS] Register token action succeeded
95 - [Dec-07 06:53 NOTIFS] Server response OK. The device_id: *** REDACTED ***
96 - [Dec-07 06:53 UTILS] trackLastActivity, activityId: Me
97 - [Dec-07 06:53 STATS] 🔵 Tracked: me_tab_accessed
98 - [Dec-07 06:53 STATS] 🔵 Tracked: support_opened, Properties: {"origin":"ME_SCREEN_HELP"}
99 - [Dec-07 06:53 UTILS] trackLastActivity, activityId: Help Screen</details>

**Comment by designsimply** [comment]
**Users affected in the last 90 days:** 734
**Limited to:** Android 8.1.0 and above
<sub>https://sentry.io/share/issue/136cee38cb79491db456071ee0c686a0/</sub>

**Comment by designsimply** [comment]
**Users affected in the last 90 days:** 782
**Limited to:** Android 8.1.0 and above
<sub>https://sentry.io/share/issue/136cee38cb79491db456071ee0c686a0/</sub>
